training/save_load_model_to_file.py

```diff
@@ -0,0 +1,30 @@
+import joblib
+
+from training.create_model import get_fitted_model
+from training.creating_dataset import create_dataframe
+
+
+def save_model_to_file(model=None, filename: str = "../saved_models/current_model.pkl"):
+    """
+    Saves a machine learning model to a file
+    :param model: machine learning model to be saved. If None, default model will be created and trained
+    :param filename: path and filename to save the model to
+    """
+
+    if model is None:
+        X = create_dataframe()
+        X.pop("book")
+        y = X.pop("level")
+
+        model = get_fitted_model(classifier_name="random_forest", X_train=X, y_train=y)
+
+    joblib.dump(model, filename)
+
+
+def load_model_from_file(filename: str = "../saved_models/current_model.pkl"):
+    """
+    Loads a machine learning model from a file
+    :param filename: path and filename of the model file to load
```

Comment:
samo "path" chyba by wystarczyło, to "and filename" mnie zmyliło i myślałam, że chcemy jakąś krotkę w tym argumencie (jeśli byś to zmieniała, to też nazwa zmiennej chyba powinna przejść na np. file_path)

Reply:
+1, bardzo słuszny komentarz. Polecam `filepath` czy coś takiego, wtedy jest jasne, że to dowolna ścieżka